project-management/src/main/java/life/qbic/projectmanagement/application/measurement/validation/MeasurementProteomicsValidator.java

```diff
@@ -166,63 +196,128 @@ ValidationResult validateInstrument(String instrument) {
           List.of(UNKNOWN_INSTRUMENT_ID.formatted(instrument)));
     }
 
+    ValidationResult validateMandatoryDataForUpdate(ProteomicsMeasurementMetadata metadata) {
+      var validation = ValidationResult.successful(1);
+      if (metadata.measurementIdentifier().isEmpty()) {
+        validation.combine(ValidationResult.withFailures(1,
+            List.of("Measurement id: missing measurement id for update")));
+      } else {
+        validation.combine(ValidationResult.successful(1));
+      }
+      if (metadata.organisationId().isBlank()) {
+        validation = validation.combine(
+            ValidationResult.withFailures(1, List.of("Organisation: missing mandatory metadata")));
+      } else {
+        validation = validation.combine(ValidationResult.successful(1));
+      }
+      if (metadata.instrumentCURI().isBlank()) {
+        validation = validation.combine(
+            ValidationResult.withFailures(1, List.of("Instrument: missing mandatory metadata")));
+      } else {
+        validation = validation.combine(ValidationResult.successful(1));
+      }
+      if (metadata.facility().isBlank()) {
+        validation = validation.combine(
+            ValidationResult.withFailures(1, List.of("Facility: missing mandatory meta;data")));
+      } else {
+        validation = validation.combine(ValidationResult.successful(1));
+      }
+      if (metadata.digestionEnzyme().isBlank()) {
+        validation = validation.combine(ValidationResult.withFailures(1,
+            List.of("Digestion Enzyme: missing mandatory metadata")));
+      } else {
+        validation = validation.combine(ValidationResult.successful(1));
+      }
+      if (metadata.digestionMethod().isBlank()) {
+        validation = validation.combine(ValidationResult.withFailures(1,
+            List.of("Digestion Method: missing mandatory metadata")));
+      } else {
+        validation = validation.combine(ValidationResult.successful(1));
+      }
+      if (metadata.enrichmentMethod().isBlank()) {
+        validation = validation.combine(ValidationResult.withFailures(1,
+            List.of("Enrichment Method: missing mandatory metadata")));
+      } else {
+        validation = validation.combine(ValidationResult.successful(1));
+      }
+      if (metadata.injectionVolume().isBlank()) {
```

Comment:
```suggestion
      if (metadata.injectionVolume().isBlank()) {
```
not mandatory anymore